src/RVRP_data.jl

```diff
@@ -34,43 +28,61 @@ mutable struct Depot
     time_windows::Vector{TimeWindow} # optional
 end
 
-mutable struct Pickup
+mutable struct PickupPoint
     id::String # If its part of a shipment, it has is own id anyway
-    index::Int # Not given in JSON. If its part of a shipment, the index of the shipment
+    index::Int # Not given in JSON. 
     location::Location
-    shipment_id::String # NULL if none
-    price_reward::Float64
-    capacity_request::Float64
     time_windows::Vector{TimeWindow} # optional
-    service_time::Float64 # optional
+    service_time::Float64 # optional; to measure access time for instance
 end
 
-mutable struct Delivery
+
+mutable struct DeliveryPoint
     id::String # If it is part of a shipment,it has is own id anyway
-    index::Int # Not given in SON. if it is part of a shipment, the index of the shipment
+    index::Int # Not given in JSON. 
     location::Location
-    shipment_id::String # NULL if none
-    price_reward::Float64
-    capacity_request::Float64
     time_windows::Vector{TimeWindow} # optional
-    service_time::Float64 # optional
+    service_time::Float64 # optional; to measure access time for instance
+end
+
+mutable struct Product
+    id::String 
+    index::Int # Not given in JSON. 
+    conflicting_product_ids::Vector{String} # if any
+    prohibited_predecessor_product_ids::Vector{String}  # if any
+end
+
+mutable struct Commodity
+    id::String 
+    index::Int # Not given in JSON. 
+    product_id::String
+    sources::Dict{PickupPoint,Float64} # available quantities; undefined if no restrictions, i.e, if available in any PickupPoint in large quantities
```

Comment:
We can not have Pointers in JSON. To avoid redundancy we need to provide ids instead